### ISSUE TITLE
Crash when adding reminder 

### ISSUE THREAD

**Issue by dzmpr** [a-zasypalov/Roar#3]
I get an app crash every time that I try to create a custom reminder.
![Screenshot_2024-04-25-09-52-56-471_com.gaoyun.roar.android.jpg](https://github.com/a-zasypalov/Roar/assets/10374594/d4082a39-7616-4716-8663-260e5f4b730f)

**Comment by a-zasypalov** [comment]
🤔 will check it on a weekend, thanks for heads up

**Comment by dzmpr** [comment]
Here stacktrace:
```text
java.lang.IllegalArgumentException: Wrong argument type for 'TEMPLATE_ID_KEY' in argument bundle. string expected.
                                                             	at androidx.navigation.NavDestination.addInDefaultArgs(NavDestination.kt:602)
                                                             	at androidx.navigation.NavController.navigate(NavController.kt:1805)
                                                             	at androidx.navigation.NavController.navigate(NavController.kt:2220)
                                                             	at androidx.navigation.NavController.navigate$default(NavController.kt:2215)
                                                             	at com.gaoyun.roar.android.MainActivity$GlobalDestinationState$1$1.invokeSuspend(MainActivity.kt:168)
                                                             	at com.gaoyun.roar.android.MainActivity$GlobalDestinationState$1$1.invoke(Unknown Source:8)
                                                             	at com.gaoyun.roar.android.MainActivity$GlobalDestinationState$1$1.invoke(Unknown Source:4)
                                                             	at kotlinx.coroutines.flow.FlowKt__TransformKt$onEach$$inlined$unsafeTransform$1$2.emit(Emitters.kt:223)
                                                             	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.emitAllImpl$FlowKt__ChannelsKt(Channels.kt:37)
                                                             	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.access$emitAllImpl$FlowKt__ChannelsKt(Channels.kt:1)
                                                             	at kotlinx.coroutines.flow.FlowKt__ChannelsKt$emitAllImpl$1.invokeSuspend(Unknown Source:14)
                                                             	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
                                                             	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
                                                             	at androidx.compose.ui.platform.AndroidUiDispatcher.performTrampolineDispatch(AndroidUiDispatcher.android.kt:81)
                                                             	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performTrampolineDispatch(AndroidUiDispatcher.android.kt:41)
                                                             	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.run(AndroidUiDispatcher.android.kt:57)
                                                             	at android.os.Handler.handleCallback(Handler.java:958)
                                                             	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                             	at android.os.Looper.loopOnce(Looper.java:224)
                                                             	at android.os.Looper.loop(Looper.java:318)
                                                             	at android.app.ActivityThread.main(ActivityThread.java:8759)
                                                             	at java.lang.reflect.Method.invoke(Native Method)
                                                             	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:561)
                                                             	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
```
Also think you should enable R8 minification :)

**Comment by a-zasypalov** [comment]
Will be released soon with version 1.1.0
